c2corg_ui/static/js/auth/authservice.js

```diff
@@ -75,32 +75,42 @@ app.Authentication.prototype.isAuthenticated = function() {
 /**
  * Checks if the current user has rights to access/edit the document.
  * TODO: rights to personal documents only for current user.
+ * @param {Array<string>} users
+ * @param {?string} imageType
  * @return {boolean}
  * @export
  */
-app.Authentication.prototype.hasEditRights = function(users) {
+app.Authentication.prototype.hasEditRights = function(users, imageType) {
```

Comment:
Probably not a bad idea. Would it be ok to write something like :

```
hasEditRIght(users, imageType) {
    isAuthenticated...
    ....
    isModerator...
   ....
    if(!imageType && users) {  checkRightsOuting() }
    else if (imageType) {  checkRightsImage() } 
}

```
because the first part of the function is always common.